socialite/src/components/Dashboard/Dashboard.css

```diff
@@ -64,4 +64,27 @@ input{
 	bottom: 25px;
 	left: 50%;
 	transform: translateX(-50%);
+}
+
+.friend{
+	position: relative;
+	background-color: rgba(0, 0, 0, .2);
+	border-radius: 10px;
+	padding: 0 1rem;
+	height: 100px;
+	text-align: center;
+	color: #00adb5;
+	margin-top: 1rem;
+}
+
+.friend:hover{
+	background-color: #3a4750;
+	cursor: pointer;
+}
+
+.friend-content{
+	position: absolute;
+	bottom: 35px;
+	left: 50%;
+	transform: translateX(-50%);
```

Comment:
Maybe put this in a separate css file Recommend.css in the Recommend directory? Would be easier to debug later.